src/bench/mempool_eviction.cpp

```diff
@@ -127,7 +127,7 @@ static void MempoolEviction(benchmark::State& state)
         AddTx(tx6_r, 1100LL, pool);
         AddTx(tx7_r, 9000LL, pool);
         pool.TrimToSize(pool.DynamicMemoryUsage() * 3 / 4);
-        pool.TrimToSize(GetVirtualTransactionSize(tx1));
+        pool.TrimToSize(GetVirtualTransactionSize(CTransaction(tx1)));
```

Comment:
Should probably move out of the hot loop (see comment above) and submit as separate pull request, since it changes the benchmark